Gemfile

```diff
@@ -113,6 +112,8 @@ source 'https://rails-assets.org' do
   gem 'rails-assets-c3', '~> 0.4.10'
   gem 'rails-assets-bootstrap-select', '~> 1.5.4'
   gem 'rails-assets-bootstrap-hover-dropdown', '~> 2.0.11'
+  gem 'rails-assets-angular', '~> 1.4.3'
+  gem 'rails-assets-angular-mocks', '~> 1.4.3'
```

Comment:
agreed, but you merged before I managed to push it sorted, so I'll do that in #3773 , ok?